packages/salesforcedx-vscode-apex/package.json

```diff
@@ -49,9 +52,89 @@
     "test": "node ./node_modules/vscode/bin/test",
     "test:unit": "node ./node_modules/vscode/bin/test"
   },
-  "activationEvents": ["workspaceContains:sfdx-project.json"],
+  "activationEvents": [
+    "workspaceContains:sfdx-project.json",
+    "onCommand:sfdx.force.test.view.showError",
+    "onView:sfdx.force.test.view"
+  ],
   "main": "./out/src",
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "test",
+          "title": "Test Viewer",
```

Comment:
We keep labels in our package.json files in corresponding package.nls.json files so that we can more easily apply localization. You can look at line 140 of this file as an example. Could you please extract the user facing strings in this file to the package.nls.json file?